.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: minimal
+
+env:
+  - IGNORE_PAT="^\[.*\]:"
+
+script:
+  - for f in text/*; do
```

Comment:
I think this check is still not appropriate. We usually put code snippets in RFC, whose line width can be 100 characters (and it is even not a hard limit by rustfmt, for example, comments will not be folded, so we can have more than 100 characters actually).

Reply:
I think code can be folded at 80 chars easily. Anyway what's you idea?

Reply:
I don't have a good solution as well. If code is limited to 80 chars, then sometimes we need to modify codes that copied from TiKV (because they are 100 chars). Or we can remove the check. Or, we can find some ways to ignore code snippets by matching ` ``` `

Reply:
Yes, let's leave it as a future work.